prov/gni/contrib/gnitest.supp

```diff
@@ -68,11 +68,41 @@
    fun:pthread_create@@GLIBC_2.2.5
    fun:_gnix_dgram_hndl_alloc
    fun:_gnix_cm_nic_alloc
+   fun:_gnix_ep_nic_init
    fun:gnix_ep_open
    fun:fi_endpoint
 }
 
 #
+# This is due to not initializing unused attr fields in struct
+# gnix_pep_sock_connreq
+#
+# {
+#    fi_connect_memcheck
+#    Memcheck:Param
+#    write(buf)
+#    obj:/lib64/libpthread-2.11.3.so
```

Comment:
Does this leak suppression depend on this version of libpthread? If not, is there a symlink like `libpthread.so` that would reference the current version?